README.adoc

```diff
@@ -71,7 +71,7 @@ include::{common-includes}/gitclone.adoc[]
 
 == Setting up MongoDB
 
-This guide uses Docker to run an instance of `MongoDB`. A multi-stage Dockerfile is provided for you. This Dockerfile uses the `mongo` image as the base image of the final stage and gathers the required configuration files. The resulting `mongo` image runs in a Docker container, and you must set up a new database for the microservice. Lastly, the truststore that's generated in the Docker image is copied from the container and placed into the Open Liberty server.
+This guide uses Docker to run an instance of MongoDB. A multi-stage Dockerfile is provided for you. This Dockerfile uses the `mongo` image as the base image of the final stage and gathers the required configuration files. The resulting `mongo` image runs in a Docker container, and you must set up a new database for the microservice. Lastly, the truststore that's generated in the Docker image is copied from the container and placed into the Open Liberty server.
```

Comment:
Paragraph should be separated into multiple lines.

Reply:
Will check with ID